src/js/administration/components/InstanceMessage.tsx

```diff
@@ -15,21 +16,17 @@ const InstanceMessageSubtitle = styled.p`
     margin: 0 0 15px;
 `;
 
+type InstanceMessageProps = {
+    /** The instance message data */
+    data: Message;
+};
+
 /**
  * Displays the instance message and provides functionality to update it
  */
-export default function InstanceMessage() {
-    const { data, isLoading } = useFetchMessage();
+export default function InstanceMessage({ data }: InstanceMessageProps) {
```

Comment:
```suggestion
export default function InstanceMessage({ message }: InstanceMessageProps) {
```
or other more descriptive name